internal/controllers/certmanager.go

```diff
@@ -122,6 +122,45 @@ func (r *Reconciler) setupTLS(ctx context.Context, cr *model.CryostatInstance) (
 		return nil, err
 	}
 
+	secret, err := r.GetCertificateSecret(ctx, caCert)
+	if err != nil {
+		return nil, err
+	}
+	// Copy Cryostat CA secret in each target namespace
+	for _, ns := range cr.TargetNamespaces {
+		if ns != cr.InstallNamespace {
+			namespaceSecret := &corev1.Secret{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      caCert.Spec.SecretName,
+					Namespace: ns,
+				},
+				Type: secret.Type,
+			}
+			err = r.createOrUpdateSecret(ctx, namespaceSecret, cr.Object, func() error {
+				namespaceSecret.Data = secret.Data
```

Comment:
Could you change this to copy only the `corev1.TLSCertKey` key? You'll need to check if `namespaceSecret.Data == nil` and initialize `namespaceSecret.Data` to an empty map if so.